homeassistant/components/panasonic_viera/__init__.py

```diff
@@ -46,6 +55,13 @@
 
 PLATFORMS = [MEDIA_PLAYER_DOMAIN]
 
+URL_EVENT_0_DMR = "dmr/event_0"
+URL_EVENT_0_NRC = "nrc/event_0"
+
+UPNP_EVENTS = [URL_EVENT_0_DMR, URL_EVENT_0_NRC]
+
+MAP_APP_NAME = {"platinum": "Browser", "Amazon": "Prime Video"}
```

Comment:
This is only provisory for correcting some app names ("Browser" is reported as "platinum" for some reason). In the future, the app display names will be retrieved directly from the TV.